package.json

```diff
@@ -150,10 +156,12 @@
     "remark-rehype": "^2.0.0",
     "remark-stringify": "^3.0.1",
     "semaphore": "^1.0.5",
-    "slate": "^0.21.0",
-    "slate-edit-list": "^0.7.1",
-    "slate-edit-table": "^0.10.1",
-    "slate-soft-break": "^0.3.0",
+    "slate": "^0.25.0",
+    "slate-edit-list": "^0.8.0",
```

Comment:
`warning "slate-edit-list@0.8.0" has incorrect peer dependency "slate@^0.23.0".`

Reply:
Yeah, I think that's just where `slate-edit-list` is at, it's the latest version.

Reply:
They do use `Block.create()` here: https://github.com/GitbookIO/slate-edit-list/tree/master/lib/changes, in `decreaseItemDepth.js` and `increaseItemDepth.js`, I don't know if that could present a problem at all.

Reply:
https://github.com/GitbookIO/slate-edit-list/blob/2ab5c0a581c71890f351069658580e620d72686f/lib/changes/decreaseItemDepth.js#L41-L45
https://github.com/GitbookIO/slate-edit-list/blob/2ab5c0a581c71890f351069658580e620d72686f/lib/changes/increaseItemDepth.js#L57-L61

Reply:
They're manually inserting chlidren, so those created blocks will never have no children. Not an issue. Thanks for checking their source!